src/mpris.cpp

```diff
@@ -17,8 +17,75 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
  */
 
+#include "mpris_p.h"
 #include "mpris.h"
 
 using namespace Amber;
 
 Mpris::Mpris() {}
+
+Mpris::LoopStatus MprisPrivate::LoopStatusStringToEnum(const QString &value, bool *ok) {
```

Comment:
Nit: method starting braces to their own lines. Applies to later ones too.

Also pondering the naming a bit, e.g. stringToLoopStatus() style would drop a few characters with the sort of redundant enum.

Reply:
Brackets and naming changed.